lighthouse-core/scripts/update-flow-fixtures.js

```diff
@@ -45,49 +50,92 @@ async function waitForImagesToLoad(page) {
   }, TIMEOUT);
 }
 
-(async () => {
+/**
+ * @param {LH.Result.MeasureEntry[]} timings
+ */
+function normalizeTimingEntries(timings) {
```

Comment:
Let's move this to `assetSaver` under a function named `normalizeArtifacts` or `normalizeArtifactsForTesting` (and use it in `lighthouse-core/scripts/update-report-fixtures.js` too)